docker-images/prometheus/cmd/prom-wrapper/receivers.go

```diff
@@ -226,8 +230,15 @@ For more details, please refer to the service dashboard: %s`, firingBodyTemplate
 				}
 			}
 
-			priority := notifier.Opsgenie.Priority
-			tags := notifier.Opsgenie.Tags
+			priority := priorityTemplateDefault
```

Comment:
So sorry I didn't bring this up earlier but I just realized - each notifier can only target a specific alert level. Can we just set the priority based on the value of `alert.Level` instead?

```suggestion
			var priority string
			if alert.Level == "warning" {
				priority = "P2"
			} else {
				priority = "P1"
			}
```

Tags template looks great though!

Reply:
PTAL, updated